zio-kafka-test-utils/src/main/scala/zio/kafka/KafkaTestUtils.scala

```diff
@@ -169,6 +171,29 @@ object KafkaTestUtils {
         )
       )
 
+  def sslAdminSettings: ZIO[Kafka, Nothing, AdminClientSettings] =
+    ZIO
+      .serviceWith[Kafka](_.bootstrapServers)
+      .flatMap(bootstrap =>
+        ZIO.attempt {
+          val trustStorePath = Paths.get(Kafka.getClass.getResource("/truststore/kafka.truststore.jks").toURI).toFile
+          val keyStorePath   = Paths.get(Kafka.getClass.getResource("/keystore/kafka.keystore.jks").toURI).toFile
```

Comment:
🟢 Why `Kafka.getClass` and not just `this.getClass`? 🤔
```suggestion
        ZIO.attempt {
          val trustStorePath = Paths.get(this.getClass.getResource("/truststore/kafka.truststore.jks").toURI).toFile
          val keyStorePath   = Paths.get(this.getClass.getResource("/keystore/kafka.keystore.jks").toURI).toFile
```

Reply:
Fixed